extensions/2.0/Khronos/KHR_audio/README.md

```diff
@@ -0,0 +1,293 @@
+            "rolloffFactor": 0.8
+          }
+        }
+      ]
+    }
+  },
+  "scenes": [
+    {
+      "name": "Default Scene",
+      "extensions": {
+        "KHR_audio": {
+          "emitters": [0]
+        }
+      }
+    }
+  ],
+  "nodes": [
+    {
+      "name": "Duck",
+      "translation": [1.0, 2.0, 3.0],
+      "extensions": {
+        "KHR_audio": {
+          "emitter": 1
+        }
+      }
+    }
+  ]
+}
+```
+
+## glTF Schema Updates
+
+This extension consists of two primary data structures: Audio Sources and Audio Emitters. Both sources and emitters are defined on an `KHR_audio` object added to the `extensions`
+ object on the document root.
+
+The extension must be added to the file's `extensionsUsed` array and because it is optional, it does not need to be added to the `extensionsRequired` array.
+
+#### Example:
+
+```json
+{
+    "asset": {
+        "version": "2.0"
+    }
+    "extensionsUsed" : [
+        "KHR_audio"
+    ],
+    "scenes": [...],
+    "nodes": [...],
+    "extensions": {
+        "KHR_audio": {
+            "sources": [...],
+            "emitters": [...]
+        }
+    }
+}
+```
+
+### Audio Sources
+
+Audio source objects define audio data to be used in audio emitters. Multiple audio emitters may use the same audio source.
+
+Audio sources can store their data in either a buffer view or reference an external file via uri.
+
+When storing audio data in a buffer view, the `mimeType` field must be specified. Currently the only supported mime type is `audio/mpeg`.
+
+#### `bufferView`
+
+The index of the bufferView that contains the audio data. Use this instead of the audio source's uri property.
+
+#### `mimeType`
+
+The audio's MIME type. Required if `bufferView` is defined. Unless specified by another extension, the only supported mimeType is `audio/mpeg`.
+
+#### `uri`
+
+The uri of the audio file. Relative paths are relative to the glTF file.
+
+#### MP3 Audio Format
+
+Provides a space efficient format that can also be tuned to satisfy audio engineers.
+
+The MPEG3 Audio Format is commonly available and freely licensed.
+
+### Audio Emitter
+
+#### `type`
+
+Specifies the audio emitter type.
+
+- `positional` Positional audio emitters. Using sound cones, the orientation is `+Z` having the same front side for a [glTF asset](https://www.khronos.org/registry/glTF/specs/2.0/glTF-2.0.html#coordinate-system-and-units). 
+- `global ` Global audio emitters are not affected by the position of audio listeners. `coneInnerAngle`, `coneOuterAngle`, `coneOuterGain`, `distanceModel`, `maxDistance`, `refDistance`, and `rolloffFactor` should all be ignored when set.
+
+#### `gain`
+
+Unitless multiplier against original source volume for determining emitter loudness.
+
+#### `loop`
+
+Whether or not to loop the specified audio clip when finished.
+
+#### `playing`
+
+Whether or not the specified audio clip is playing. Setting this property `true` will set the audio clip to play on load (autoplay).
```

Comment:
Is there a particular reason for naming this like a state ("playing") vs. naming it as what it does ("autoplay")? Seems that hints at possible implementation details ("changing this should change play state") but that isn't mentioned.

Reply:
This is a bigger discussion about how animation, audio, etc. should be treated on load. Animations are currently just keyframe data and it's up to the implementation to figure out how to play the animations. https://www.khronos.org/registry/glTF/specs/2.0/glTF-2.0.html#animations

So this begs the question if `playing` or `autoPlay` or even `loop` should be included in the spec.

@najadojo @bghgary this also goes against the direction of the MSFT_audio_emitter where there are ways to specify runtime behavior.

Maybe runtime behavior should be left out of this spec and implemented in another?

Reply:
During the OMI glTF meeting we discussed this a bit. I think synchronizing audio with an animation should also be made part of another spec. AFAIK synchronizing audio with animations isn't necessarily the best way to deal with this scenario. We should talk more about `autoPlay` and `loop` though. Should this be included in the spec? Should we go into more depth on playing/mixing audio? It'd be good to get some feedback from others on this.

Reply:
I think, at a minimum, we need to define both how the glTF carries the audio payload (including metadata) and how to play the audio, whether that is one more specs. If this spec only defines the former and that's all we define, it will be hard to know if the spec works and it will be hard to demo.

Reply:
Would you agree that the options are
- adding info about play behaviour to the audio data itself
- connecting audio and animations in some way (either from audio to animation or from animation to audio)
or do you have another one in mind? 

I agree that without any means of knowing how the audio is relevant to the scene, viewers won't be able to do anything with it - e.g. at a minimum I think tools like model-viewer should have a way to infer what to do with a file that has multiple animation clips and multiple audio assets (could be 1:1, could be different). A counter-argument would be saying "well for this case, if there's one animation and one audio clip that will be played back, everything else is undefined" (still allowing for cases such as [this](https://songsofcultures.com/en/ar-collection)) but I'm not a huge fan of that.

Reply:
During the OMI glTF meeting we agreed that playing behavior (`audioPlay` and `loop`) should be in this spec to define the minimum behavior to play sounds. However, connecting audio and animations should be delegated to an extension like https://github.com/KhronosGroup/glTF/pull/2147

Reply:
I think we need to think about the corresponding spec(s) that defines how the audio will play before completing this spec. `KHR_animation_pointer` spec will be able to cover some things (like modifying parameters for looping sounds), but it's probably not enough (e.g. one-shot sounds triggered from an animation).